source/core/rtc_adapter/VideoSendAdapter.h

```diff
@@ -29,7 +29,8 @@ namespace rtc_adapter {
 
 class VideoSendAdapterImpl : public VideoSendAdapter,
                              public webrtc::Transport,
-                             public webrtc::RtcpIntraFrameObserver {
+                             public webrtc::RtcpIntraFrameObserver,
```

Comment:
Provide an explicit override declaration of   OnReceivedIntraFrameRequest() here.